bin/alias_schemas

```diff
@@ -57,7 +57,7 @@ def main(aliases_path, base_dir):
                 fname = source.name.replace(source_doctype, dest_doctype)
                 dest = dest_dir / fname
 
-                print(f"Aliasing {dest} to {source}")
+                print(f"Aliasing {source} to {dest}")
```

Comment:
Unrelated bug fix.

Reply:
Hah, I remember writing this and being undecided about the order.

Reply:
I was definitely interpreting this as basically meaning "I've copied X source file to Y destination". But I can now see how you might think about aliasing the opposite way.

Reply:
Yeah, probably just needs a bit more explanatory text.